### PR TITLE
Sign Up Button on Splash Page

### PR DESCRIPTION
**Pull request by PuneetDabral**
sir please check this...

### CODE REVIEW

File: src/SplashScreen/SplashNav/SplashNav.tsx [hunk 1]
```diff
@@ -48,9 +48,9 @@ function SplashNav() {
             <NavItem>
               <Login>Log in</Login>
             </NavItem>
```
Comment (scottjr632) [inline src/SplashScreen/SplashNav/SplashNav.tsx:50]: Please also remove the "Log In" button.
```suggestion

```

**Review by scottjr632 — CHANGES_REQUESTED** [review]
Please remove the "Login In" button as well. It is above the "Sign Up" button.

**Comment by PuneetDabral** [comment]
ok sir now check it and merge it...thank you very much sir i learn lot  from you.....also give me some more issue so i learn more & more

**Review by scottjr632 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**